dev/example-config.json

```diff
@@ -60,15 +64,28 @@
         }
       },
       "backend": {
-        "backend_wide_key_consumer_cert": "config/backend/distributey_serviceX_key_consumer.crt"
-      },
-      "vault_auth_jwt_path": "jwt-dev",
-      "vault_default_role": "salesforce-dev",
-      "vault_transit_path": "transit-dev"
+        "backend_wide_key_consumer_cert": "config/backend/distributey_serviceX_key_consumer.crt",
+        "VAULT": {
+          "cacert": "config/myCA.crt",
+          "mtls_client_cert": "config/mtls_auth.crt",
+          "mtls_client_key": "config/mtls_auth.key",
+          "url": "https://127.0.0.1:8300",
+          "namespace": "",
+          "auth_jwt_path": "jwt-dev",
+          "default_role": "salesforce-dev",
+          "transit_path": "transit-dev"
+        }
+      } 
     }
   },
-  "VAULT_CACERT": "config/myCA.crt",
-  "VAULT_MTLS_CLIENT_CERT": "config/mtls_auth.crt",
-  "VAULT_MTLS_CLIENT_KEY": "config/mtls_auth.key",
-  "VAULT_URL": "https://vault:8300"
```

Comment:
This name "vault" can not be resolved when using the "dev mode" and the dev server:
```
[2021-12-20 15:38:40,486] distributey {/home/andi/Downloads/distributey/distributey/vault_backend.py:101} ERROR - tenant: monitoring, origin: 127.0.0.1, ua: curl/7.68.0 - Failed to authenticate against Vault: HTTPSConnectionPool(host='vault', port=8300): Max retries exceeded with url: /v1/auth/jwt/login (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f6f89956820>: Failed to establish a new connection: [Errno -2] Name or service not known'))
```

This is why I changed it to "127.0.0.1" in a recent commit in the example config.

Reply:
Resolving `vault` is indeed an issue in dev mode, which is why I made the following comment in the docs: https://github.com/p15r/distributey/blob/9befb515c7ef64ce91a1366b610f28b936c58ecd/docs/usage.md?plain=1#L49

I decided not to switch to `127.0.0.1` by default in the config, because that wont work if we added a second Vault instances. Probably, it is more pragmatic though to switch to `127.0.0.1` by default...